build.zig

```diff
@@ -77,6 +77,20 @@ pub fn build(b: *std.Build) !void {
     const run_step = b.step("run", "Run the app");
     run_step.dependOn(&run_cmd.step);
 
+    const tests = b.addTest(.{
+        .name = "slimytest",
+        .root_source_file = b.path("src/main.zig"),
+        .target = target,
+        .optimize = optimize,
+        .single_threaded = singlethread,
+        .strip = strip,
+    });
```

Comment:
No need for a `name`, just leave it empty :)
And tests should never be stripped, so that line can go too